datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/logs/{check_name}/manifest.json

```
@@ -1,35 +1,40 @@
 {{
-  "display_name": "{integration_name}",
-  "maintainer": "{email}",
-  "manifest_version": "1.0.0",
-  "name": "{check_name}",
-  "metric_prefix": "{check_name}.",
-  "metric_to_check": "",
-  "creates_events": false,
-  "short_description": "",
-  "guid": "{guid}",
-  "support": "{support_type}",
-  "supported_os": [
-    "linux",
-    "mac_os",
-    "windows"
-  ],
-  "public_title": "{integration_name}",
-  "categories": [
-    "log collection"
-  ],
-  "type": "check",
-  "is_public": false,
-  "integration_id": "{integration_id}",{pricing_plan}{terms}{author_info}
+  "manifest_version": "2.0.0",
+  "app_uuid": "{guid}",
+  "app_id": "{integration_id}",
+  "display_on_public_website": false,
+  "tile": {{
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "",
+    "title": "{integration_name}",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
```

```suggestion
      "Private :: Do Not Upload",
      "Supported OS::Linux",
```
From https://github.com/DataDog/integrations-core/pull/12556
Same for the other templates

---

That's a Python thing in a different file but sure

---

Oops my bad just noticed it should be in `pyproject.toml` not `manifest.json` 🤦